### ISSUE TITLE
[Assistive tech and device support] Keyboard navigation order isn't meaningful or sequential. (11.08.1)

### ISSUE THREAD

**Issue by taylorkaren** [department-of-veterans-affairs/va-forms-system-core#249]
@shiragoodman commented on [Thu Jan 06 2022](https://github.com/department-of-veterans-affairs/va.gov-team/issues/34872)

#### VFS team name
Claims & Appeals

#### VFS product name
HLR V2

#### Point of Contact/Reviewers
Angela Fowler (@AngelaFowler82) - Accessibility

---

### Platform Issue
Keyboard navigation order isn't meaningful or sequential.

### Issue Details
When filling out the application, the progress indicater can be accessed using the tab key. This could fool keyboard users into thinking it is clickable.

### VA.gov Experience Standard
[Category Number 11, Issue Number 08](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 2.4.3_A

---

### Platform Recommendation
Remove the progress indicater from the tab order.

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket. 

---

@shiragoodman commented on [Thu Jan 06 2022](https://github.com/department-of-veterans-affairs/va.gov-team/issues/34872#issuecomment-1006825283)

Follow up on this as it may be a progress bar issue.

---

@Mottie commented on [Fri Jan 07 2022](https://github.com/department-of-veterans-affairs/va.gov-team/issues/34872#issuecomment-1007795751)

I have reported this issue to the design system team and plan on following up on how to address this problem.

---

@Mottie commented on [Mon Jan 10 2022](https://github.com/department-of-veterans-affairs/va.gov-team/issues/34872#issuecomment-1008977125)

Related to https://github.com/department-of-veterans-affairs/va.gov-team/issues/21138
CC: @caw310 

---

@shiragoodman commented on [Mon Mar 07 2022](https://github.com/department-of-veterans-affairs/va.gov-team/issues/34872#issuecomment-1061151527)

hey @Mottie  - following up on this. This ticket is still assigned to Claims and Appeals. In your comments above, are you stating the ticket should be reassigned to Design System?

---

@Mottie commented on [Mon Mar 07 2022](https://github.com/department-of-veterans-affairs/va.gov-team/issues/34872#issuecomment-1061153770)

Yes, reassign please

---

@shiragoodman commented on [Mon Mar 07 2022](https://github.com/department-of-veterans-affairs/va.gov-team/issues/34872#issuecomment-1061156217)

@caw310 @k80bowman This ticket has been reassigned from Claims and Appeals team to Design System team. Please let me know if you have any questions.

**Comment by bjmccotter7192** [comment]
Commented in the original ticket to have the tabIndex altered/removed from the va-segmented-progress-bar to eliminate the ability to navigate to it using the keyboard.

They would like an accessibility expert to verify but I don't think we will need to do any work in the `Forms System` for this ticket.

@taylorkaren

**Comment by taylorkaren** [comment]
Also commented in the original ticket: There is a hard coded tab index in the component, so bypassing it doesn't do any good. Remove the tab index or change it to -1 so it is not navigable by keyboard but would be navigable by JS. BJ can put a PR in for the Design System Team to review and have an accessibility specialist test the change.

**Comment by bjmccotter7192** [comment]
[Link to PR](https://github.com/department-of-veterans-affairs/component-library/pull/418)

**Comment by taylorkaren** [comment]
Comments from the original ticket (https://github.com/department-of-veterans-affairs/va.gov-team/issues/34872):

bkjohnson commented 15 hours ago • 
@joshkimux, should we remove tabindex completely, or should it have tabindex="-1" for any kind of programmatic focus?

@joshkimux
Member
joshkimux commented 14 hours ago
@bkjohnson given it isn't interactive and that there are written steps, I don't see the need of it ever needing programmatic focus. Removing the tabindex completely should do the trick 😄
